spanner/spansql/parser_test.go

```diff
@@ -656,6 +659,9 @@ func TestParseFailures(t *testing.T) {
 		// Found by fuzzing.
 		// https://github.com/googleapis/google-cloud-go/issues/2196
 		{query, `/*/*/`, "invalid comment termination"},
+		// Spanner does not have unary plus.
+		// This should also not cause a panic.
+		{expr, `+(Y - Z)`, "unary plus"},
```

Comment:
See above. It is undocumented, but it is supported and will not cause a parse error on Cloud Spanner.